src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -613,7 +621,7 @@ public override BoundNode VisitArrayAccess(BoundArrayAccess node)
 
             if (node.Indices.Length != 1)
             {
-                return base.VisitArrayAccess(node);
+                return base.VisitArrayAccess(node)!;
```

Comment:
Is the base returning a nullable return value? Could we change it to return not-nullable instead?

Reply:
Yes.  No, because many overrides do return null.

---
In reply to: [386594458](https://github.com/dotnet/roslyn/pull/42068#discussion_r386594458) [](ancestors = 386594458)